src/main/java/winstone/HostConfiguration.java

```diff
@@ -112,12 +109,12 @@ public HostConfiguration(Server server, String hostname, ClassLoader commonLibCL
                 this.webapps.keySet() + "");
     }
 
-    private void loadBuiltinMimeTypes() {
+    private void loadBuiltinMimeTypes(Server server) {
```

Comment:
Shadowing a field by the same name is usually considered a bad coding practice and reported as a warning by some static analysis tools. Simpler to get rid of this parameter and directly reference the field from this instance method.